src/patternfly/components/Popover/styles.scss

```diff
@@ -0,0 +1,280 @@
+//     border-right-color: transparent;
+//     border-left-color: transparent;
+//   }
+
+//   &.pf-m-right &__container {
+//     bottom: 50%;
+//     left: calc(100% + var(--pf-c-popover__arrow--Size) + 10px);
+//     justify-content: flex-start;
+//     transform: translateY(50%);
+//   }
+
+//   &.pf-m-right &__arrow {
+//     bottom: calc(50% - var(--pf-c-popover__arrow--Size));
+//     left: calc(-1 * var(--pf-c-popover__arrow--Size));
+//     border-width: var(--pf-c-popover__arrow--Size) var(--pf-c-popover__arrow--Size) var(--pf-c-popover__arrow--Size) 0;
+//     border-top-color: transparent;
+//     border-bottom-color: transparent;
+//     border-left-color: transparent;
+//   }
+
+//   &.pf-m-compressed {
+//     --pf-c-popover__arrow--Size: var(--pf-global--arrow--BorderWidth-sm);
+//     --pf-c-popover__content--PaddingTop: var(--pf-global--spacer--sm);
+//     --pf-c-popover__content--PaddingRight: var(--pf-global--spacer--sm);
+//     --pf-c-popover__content--PaddingBottom: var(--pf-global--spacer--md);
+//     --pf-c-popover__content--PaddingLeft: var(--pf-global--spacer--md);
+//   }
+
+//   &.pf-m-inverted {
+//     --pf-c-popover__arrow--DropShadow: none;
+//     --pf-c-popover__arrow--BorderColor: var(--pf-global--Color--dark-200);
+//     --pf-c-popover__content--BackgroundColor: var(--pf-global--Color--dark-200);
+
+//     .pf-c-popover__container {
+//       --pf-c-popover__content--Color: var(--pf-global--Color--light-100);
+//     }
+
+//     .pf-c-popover__header-text,
+//     .pf-c-popover__body,
+//     .pf-c-button.pf-m-action {
+//       color: var(--pf-global--Color--light-100);
+//     }
+//   }
+// }
+
+.pf-c-popover {
+  // Component variables
+  --pf-c-popover--MinWidth: 100px;
+  --pf-c-popover--MaxWidth: 300px;
+  --pf-c-popover--BoxShadow: var(--pf-global--BoxShadow);
+  --pf-c-popover--Margin: var(--pf-global--arrow--size);
+
+  // Arrow variables
+  --pf-c-popover__arrow--Width: var(--pf-global--arrow--size);
+  --pf-c-popover__arrow--Height: var(--pf-global--arrow--size);
+  --pf-c-popover__arrow--Margin: var(--pf-global--arrow--size) / 2;
+  --pf-c-popover__arrow--BoxShadow: var(--pf-global--BoxShadow);
+  --pf-c-popover__arrow--BackgroundColor: var(--pf-global--BackgroundColor--100);
+
+  $half-size: $pf-global--arrow--size / 2;
+
+  position: relative;
+  min-width: var(--pf-c-popover--MinWidth);
+  max-width: var(--pf-c-popover--MaxWidth);
+  box-shadow: var(--pf-c-popover--BoxShadow);
+
+  &__content {
+    min-height: 0;
+    box-shadow: none;
+  }
+
+  &__arrow {
+    position: absolute;
+    width: var(--pf-c-popover__arrow--Width);
+    height: var(--pf-c-popover__arrow--Height);
+    margin: var(--pf-c-popover__arrow--Margin);
+    pointer-events: none;
+    background-color: var(--pf-c-popover__arrow--BackgroundColor);
+    box-shadow: var(--pf-c-popover__arrow--BoxShadow);
+    transform: rotate(45deg);
+  }
+
+  &.pf-m-top {
+    margin-bottom: var(--pf-c-popover--Margin);
```

Comment:
I see the need for a single size declaration, but this name breaks the system. Therefore I'd just use `--pf-c-popover__arrow--Width` and write a good comment explaining whats happening here.